modules/core/src/smithy4s/schema/SchemaVisitor.scala

```diff
@@ -43,3 +43,21 @@ trait SchemaVisitor[F[_]] extends (Schema ~> F) {
     case LazySchema(make) => lazily(make)
   }
 }
+
+object SchemaVisitor {
+
+  abstract class Default[F[_]] extends SchemaVisitor[F]{
```

Comment:
@daddykotex, I've added this, which will help [this](https://github.com/disneystreaming/smithy4s/pull/296/files#r916620199)

Reply:
thanks, it was on my todo for sure. a replacement for the stubschematic was inevitable